bolts-tasks/src/main/java/bolts/ExecutorException.java

```diff
@@ -0,0 +1,13 @@
+package bolts;
+
+/**
+ * This is a wrapper class for emphasizing that task failed due to bad Executor, rather than the continuation block it self.
+ * <p/>
+ * Created by yunarta on 22/12/15.
```

Comment:
Could you remove this? We don't record authorship on class JavaDocs, but through git blame/commit history.